xla/backends/cpu/runtime/onednn_thunk.cc

```diff
@@ -0,0 +1,150 @@
+/* Copyright 2024 The OpenXLA Authors.
```

Comment:
The copyright header should be similar to other existing files. We are missing some newlines.
Please add a newline after this line.
```suggestion
/* Copyright 2024 The OpenXLA Authors.

```
Same for other files.